### PR TITLE
Update environment-variables.md

### LINKED ISSUE #120: Incorrect description of GITHUB_WORKSPACE environment variable
### What feature or product is affected?

GitHub Actions. Specifically the description of `GITHUB_WORKSPACE` at https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

We discoved this when writing https://github.com/github/codeql-action/pull/54 and trying to set up a self-hosted runner.

### What is the new or expected behavior?

The `GITHUB_WORKSPACE` environment variable contains the checkout path of the repository.

### How is the old or inaccurate behavior currently documented?

The current documentation says "The workspace directory contains a subdirectory with a copy of your repository ..." but this is misleading or wrong. It's unclear whether "The workspace directory" is referring to `RUNNER_WORKSPACE` (which exists but is undocumented) or `GITHUB_WORKSPACE` (the variable being documented). If the former then it's technically correct but confusing as that variable is otherwise undocumented. If the latter then it is incorrect.

To fix this, just change "The workspace directory contains a subdirectory with a copy of your repository ..." to "The workspace directory a copy of your repository ..." and leave the rest unchanged.

### Who does this affect?

Some small percentage of actions developers. In particular those who aim for their actions to work on self-hosted runners where the directory structure is more important.

### What is the impact to users?

Largely just momentary confusion because by looking at the values of all the environment variables you can easily tell what is what and spot the error.

### Content strategy and implementation

| Item | Core | DSP | Ecosystem/API |
| --- | --- | --- | --- |
| Unified content strategy plan | | | |

**Please link any in-flight PRs to this issue.**

### PR DESCRIPTION
**Pull request by casals**
### Why:

Fixes #120 . 

### What's being changed:

I changed the proposed solution from "The workspace directory a copy of your repository" to "The workspace directory - a copy of your repository" for better readability.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Amazing @casals! Thanks for another one 🔥 I'll let the @github/docs-content-ecosystem team know ⚡

File: content/actions/reference/environment-variables.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ We strongly recommend that actions use environment variables to access the files
 | `GITHUB_REPOSITORY` | The owner and repository name. For example, `octocat/Hello-World`. |
 | `GITHUB_EVENT_NAME` | The name of the webhook event that triggered the workflow. |
 | `GITHUB_EVENT_PATH` | The path of the file with the complete webhook event payload. For example, `/github/workflow/event.json`. |
-| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path. The workspace directory contains a subdirectory with a copy of your repository if your workflow uses the [actions/checkout](https://github.com/actions/checkout) action. If you don't use the `actions/checkout` action, the directory will be empty. For example, `/home/runner/work/my-repo-name/my-repo-name`. |
+| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path. The workspace directory - a copy of your repository if your workflow uses the [actions/checkout](https://github.com/actions/checkout) action. If you don't use the `actions/checkout` action, the directory will be empty. For example, `/home/runner/work/my-repo-name/my-repo-name`. |
```
Comment (lucascosti) [inline content/actions/reference/environment-variables.md:52]: Thanks a lot for this change, @casals! 💝

I think we should just use `is` for this:

```suggestion
| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path. The workspace directory is a copy of your repository if your workflow uses the [actions/checkout](https://github.com/actions/checkout) action. If you don't use the `actions/checkout` action, the directory will be empty. For example, `/home/runner/work/my-repo-name/my-repo-name`. |
```

Test workflow runs to confirm the described behavior:

* [without checkout](https://github.com/lucascosti/actions-public-playground/runs/1224377576?check_suite_focus=true#step:4:4) (empty dir)
* [with checkout](https://github.com/lucascosti/actions-public-playground/runs/1224381411?check_suite_focus=true#step:5:5)

Comment (casals) [reply content/actions/reference/environment-variables.md]: Done!

**Review by lucascosti — APPROVED** [review]
Thanks, @casals! I'll get this merged in! 🎉

**Pull request merged**